### ISSUE TITLE
Pre-gzip (zopfli) and pre-brotli versions of static files for Nginx

### ISSUE THREAD

**Issue by cthoyt** [scolby33/OCSPdash#32]
What about using this: http://flask-assets.readthedocs.io/en/latest/?

**Comment by scolby33** [comment]
That's interesting! I *don't* like how it'll make the first request for each asset slow, but I *do* like how simple and integrated it'd be.

**Comment by cthoyt** [comment]
you can always make your life awful and put stuff like `yarn` in there and have more build tools

**Comment by scolby33** [comment]
Or a makefile.

For the pre-compressed stuff, it's something along the lines of `RUN gzip **/*` in the Dockerfile, though.

**Comment by cthoyt** [comment]
Good point, docker can the build more reproducible so it's okay if it gets a *bit* more complicated
